fe/model_utils.py

```diff
@@ -53,12 +54,24 @@ def is_hydrogen(i):
     return masses
 
 
-# def generate_openmm_topology(objs, host_coords, out_filename):
-def generate_openmm_topology(objs, coords, out_filename=None):
+def image_molecule(mol_coords, box):
+    """Given a set of coordinates for a single molecule and a box will return
+    the coordinates wrapped into the periodic box.
+    """
+    centroid = np.mean(mol_coords, axis=0)
+    box_diag = np.diag(box)
+    new_center = box_diag * np.floor(centroid / box_diag)
+    return mol_coords - new_center
+
+
+def generate_openmm_topology(objs, coords, out_filename=None, box=None):
     rd_mols = []
 
     mol_sizes = []
+    # Convert nm into angstrom
     coords = coords * 10
+    if box is not None:
+        box = box * 10
```

Comment:
nit: not necessary for this PR, but would it make sense to introduce `box_angstroms` instead of  overwriting?